cloud/edgecontroller/pkg/devicecontroller/utils/kubeconfig.go

```diff
@@ -0,0 +1,21 @@
+package utils
```

Comment:
Can we merge this file into kubeclient.go for decreasing files?

Reply:
Done, thanks.